CONTRIBUTING.md

```diff
@@ -0,0 +1,284 @@
+# Contributing Guide
+
+## Welcome to Percona Server for MySQL!
+
+We're excited to have you join the Percona community and participate in keeping open-source open.
+
+### Why Your Contribution Matters?
+
+Main driving force of open-source project is community. We, as developers team, highly appreciate any contributions.
+
+It may be advisable to initially submit bug fixes to the upstream project that will eventually become part of the next
+release of Percona. Alternatively, Percona provides expedited response times, transparent development processes, and
+streamlined contribution acceptance within a public repository.  
+You can contribute in one of the following ways:
+
+* Submit a bug report or a feature
+   request in [ our public Jira ](https://jira.percona.com/projects/PS/issues).
+* Submit a pull request (PR) with the code patch.
+* Contribute to [the documentation](https://github.com/percona/psmysql-docs/blob/innovation-release/contributing.md)
+* Reach us on our [ Forums ](https://forums.percona.com/c/mysql-mariadb)
+
+This document describes the workflow for submitting pull requests.
+By contributing, you agree to the [Percona Community code of conduct](https://percona.community/contribute/coc/). 
+
+### Steps of the workflow
+
+Steps of the workflow for submitting pull request are:
+
+1. Creating Jira issue
+2. Forking of Percona repository
```

Comment:
Should match the headers - you should add
Working with source code as Number 2.

Reply:
Fixed